insta-clone/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import moment from 'moment';
+import './CommentSection.css';
+import Comment from './Comment'
+
+class CommentSection extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            comments:[],
+            newComment: "",
+            likes:0,
+            liked: false
+        }
+    }
+    componentDidMount(){
+        this.setState({
+        comments:this.props.comArray, 
+        likes:this.props.likes 
+        });
+    }
+
+    changeHandler = event => {
+        this.setState({
+            newComment: event.target.value
+        })
+    }
+    addNewComment = event => {
+        event.preventDefault();
+        const username="Berto";
+        const text= this.state.newComment;
+        this.setState( {
+            comments:[...this.state.comments, {username, text}],
+            newComment:""
+        },
+        )
+    }
+
+    miLikey = event => {
+        this.setState({liked: !this.state.liked})
+    }
+
+    render(){
+        return(
+            <div className="comment-section">
```

Comment:
Use correct spacing for elements for better readability. It is good practice to indent 2 spaces or tab with nested tags